extensions/ql-vscode/test/vscode-tests/cli-integration/databases/database-fetcher.test.ts

```diff
@@ -41,6 +42,8 @@ describe("database-fetcher", () => {
 
   afterEach(async () => {
     await cleanDatabases(databaseManager);
+    await remove(storagePath);
+    await ensureDir(storagePath);
```

Comment:
Should this be in `beforeEach`? Create the path at the start of the test, remove it in the end.

Reply:
It seems like the `ensureDir` call here is not necessary since [the directory is created in the database fetcher](https://github.com/github/vscode-codeql/blob/636f8f1b5f8a12dfb331dd8036392eaf3b47ecb8/extensions/ql-vscode/src/databases/database-fetcher.ts#L359), so I've removed it.